app/assets/javascripts/discourse/app/helpers/directory-item-user-field-value.js

```diff
@@ -0,0 +1,16 @@
+import { htmlSafe } from "@ember/template";
+import { registerUnbound } from "discourse-common/lib/helpers";
+
+export default registerUnbound(
+  "directory-item-user-field-value",
+  function (args) {
+    // Args should include key/values { item, column }
+
+    const value =
+      args.item.user && args.item.user.user_fields
+        ? args.item.user.user_fields[args.column.user_field_id]
+        : null;
+    const content = value || "-";
+    return htmlSafe(`<span class='user-field-value'>${content}</span>`);
```

Comment:
I wonder if these helpers might be better as simple components. I'm on the fence. I suspect we can try it for now.